gatsby-config.js

```diff
@@ -93,5 +94,7 @@ module.exports = {
         pathToConfigModule: "src/utils/typography",
       },
     },
+    "gatsby-plugin-now",
+    "gatsby-plugin-twitter",
```

Comment:
Why do we need this?

Reply:
It includes the JS to activate the Twitter embed